src/ray/rpc/gcs_server/gcs_rpc_client.h

```diff
@@ -45,16 +45,51 @@ class Executor {
   std::function<void(GcsRpcClient *gcs_rpc_client)> operation_;
 };
 
-// Define a void GCS RPC client method.
+/// Define a void GCS RPC client method.
+///
+/// Example:
+///   VOID_GCS_RPC_CLIENT_METHOD(
+///     ActorInfoGcsService,
+///     CreateActor,
+///     actor_info_grpc_client_,
+///     /*method_timeout_ms*/ -1,) # Default value
+///   generates
+///
+///     # Asynchronous RPC. Callback will be invoked once the RPC is replied.
+///     rpc_client_.CreateActor(request, callback, timeout_ms = -1);
+///
+///     # Synchronous RPC. The function will return once the RPC is replied.
+///     rpc_client_.SYNC_CreateActor(request, *reply, timeout_ms = -1);
```

Comment:
The `SYNC_` prefix does not match our current style guide. How about e.g. `CreateActorSync()` or `SyncCreateActor()`? Also, we can change this later but what about accepting an `absl::Duration` instead of integer millisec? It can default to absl::InfiniteDuration().

Reply:
I will go with `SyncCreateActor`. We have some code that has uppercase-prefixed name, such as STATS_... and FLAGS_... Not sure what's the standard here haha